both/helpers.js

```diff
@@ -1,322 +1,296 @@
-if(Meteor.isClient) {
-  Template.registerHelper("roundName", function(roundCode) {
+(function() { // This is a Closure
```

Comment:
Oh dear...

Here is what happened:
Through a RubyMine mishap, I did this editing  in a generated helpers.js file under .meteor/. When I discovered that, I didn't want to redo all the work, so I copied the whole file into the regular file.

Then I noticed the closure, which I didn't understand, but didn't want to remove it. I added the comment, since I found it confusing, and also I was proud to have spotted it.